index.html

```diff
@@ -459,11 +459,11 @@
 types of [=processing=] are [=appropriate=] and which are not, offloading [=privacy labour=]
 to the people using a system.
 
-In rare and highly specific cases, [=people=] should be able to [=consent=] to more
```

Comment:
The extent to which this is "rare" could be different across different categories of sites. Some groups of users might expect to give consent for purposes that would be desirable for their use of the site.